src/components/social-tile/_social-tile.scss

```diff
@@ -1,17 +1,17 @@
 .social-tile {
 
-  @media screen and (min-width: map-get($bp, 'md')) {
+  @media screen and (min-width: map-get($breakpoints, "breakpoint-3")) {
     display: flex;
   }
 
   &__icon {
-    height: 1.2em;
-    width: 1.2em;
+    height: 1.2rem;
+    width: 1.2rem;
```

Comment:
- [ ] `width: 1.2rem;` isn't in alphabetical order.
- [ ] Are `width` and `height` linked? If so, consider a local variable.